src/core/variables/EmployerLayOffFileExtensionsList.js

```diff
@@ -0,0 +1,11 @@
+import mime from 'mime';
+
+const EmployerLayOffAllowFileExtension = [
+    'xlsx',
+];
+
+const EmployerLayOffAllowFileMimeTypes = () => EmployerLayOffAllowFileExtension
+    .map(fileExt => mime.getType(fileExt));
+
+export { EmployerLayOffAllowFileExtension, EmployerLayOffAllowFileMimeTypes };
+export default EmployerLayOffAllowFileMimeTypes;
```

Comment:
Do you think it is okey to use default imports ?)

Reply:
I can remove it

Reply:
Done